galpy/potential/Potential.py

```diff
@@ -578,7 +578,7 @@ def Rzderiv(self,R,Z,phi=0.,t=0.):
         except AttributeError: #pragma: no cover
             raise PotentialError("'_Rzderiv' function not implemented for this potential")      
 
-    def normalize(self,norm,t=0.):
+    def normalize(self,norm):
```

Comment:
Why did you remove the time dependence here? I don't think it's used ever, that's true...

Reply:
yeah, 1) because I could never find a time it's used and 2) because I couldn't think of a time when a potential would be normalized except at instantiation.

Reply:
also, `normalize` wasn't passing the parameter to `Rforce`.